actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsActor.java

```diff
@@ -470,22 +471,21 @@ private void courseConsumptionMetrics(Request actorMessage) {
       }
 
       String channel = (String) rootOrgData.get(JsonKey.HASHTAGID);
-      ProjectLogger.log("Channel" + channel);
-      String responseFormat =
-          (String) cache.getData(JsonKey.CourseConsumption, courseId, periodStr);
-      if (StringUtils.isBlank(responseFormat)) {
-        responseFormat = getCourseConsumptionData(periodStr, courseId, requestObject, channel);
-        cache.setData(JsonKey.CourseConsumption, courseId, periodStr, responseFormat);
-      }
+      ProjectLogger.log(
+          "CourseMetricsActor:courseConsumptionMetrics organisation hashtag id." + channel,
```

Comment:
From:
"CourseMetricsActor:courseConsumptionMetrics organisation hashtag id."

To:
"CourseMetricsActor:courseConsumptionMetrics: Root organisation hashtag id = "